File: Módulo II/Atividade prática - módulo II/DYEGO FELIPE DE OLIVEIRA/UML_JAVA/AgenciaViagem/src/AgenciaViagem/Cliente.java
package AgenciaViagem;

public class Cliente {
	
	private String nome;
	private String cpf;
	private Destino destino;
	
	public void setNome(String nome) {
		this.nome = nome;
	}
	public String getNome() {
		return nome;
	}
	public void setCpf(String cpf) {
		this.cpf = cpf;
	}
	public String getCpf() {
		return cpf;
	}
	public void setDestino(Destino destino) {
		this.destino = destino;
	}
	public Destino getDestino() {
		return destino;
	}

}

File: Módulo III/C#/Monitoria/Monitoria30Nov/Livraria/Cliente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Livraria
{
    class Cliente
    {
        private string nome;
        private string cpf;
        private Endereco endereco;

        public string getNome()
        {
            return this.nome;
        }

        public void setNome(string nome)
        {
            this.nome = nome;
        }

        public string getCpf()
        {
            return this.cpf;
        }

        public void setCpf(string cpf)
        {
            this.cpf = cpf;
        }

        public Endereco getEndereco()
        {
            return this.endereco;
        }

        public void setEndereco(Endereco endereco)
        {
            this.endereco = endereco;
        }

    }
}
